flutter_theoplayer_sdk/flutter_theoplayer_sdk/lib/src/theoplayer_internal.dart

```diff
@@ -294,65 +477,155 @@ class THEOplayer implements EventDispatcher {
   /// * [ReadyState.have_current_data] : The player has its current frame in its buffer.
   /// * [ReadyState.have_future_data] : The player has enough data for immediate playback.
   /// * [ReadyState.have_enough_data] : The player has enough data for continuous playback.
+  ///
+  /// Deprecated("Use [readyState] instead.")
+  @Deprecated("Use [readyState] instead.")
   ReadyState getReadyState() {
+    return readyState;
+  }
+
+  /// The ready state of the player, represented by a value from the following list:
+  /// * [ReadyState.have_nothing] : The player has no information about the duration of its source.
+  /// * [ReadyState.have_metadata] : The player has information about the duration of its source.
+  /// * [ReadyState.have_current_data] : The player has its current frame in its buffer.
+  /// * [ReadyState.have_future_data] : The player has enough data for immediate playback.
+  /// * [ReadyState.have_enough_data] : The player has enough data for continuous playback.
+  ReadyState get readyState {
     return _playerState.readyState;
   }
 
   /// Whether the player is seeking.
+  /// Deprecated("Use [seeking] instead.")
+  @Deprecated("Use [seeking] instead.")
   bool isSeeking() {
+    return seeking;
+  }
+
+  /// Whether the player is seeking.
+  bool get seeking {
```

Comment:
Would it make more sense to keep the names of the boolean properties with the `is` prefix?
`player.isSeeking` / `player.isInitialized`...